templates/conf.py_t

```diff
@@ -235,7 +235,7 @@ intersphinx_mapping = {'https://docs.python.org/': None}
 {% endif %}
 
 # Adding scripts directory to path
-sys.path.insert(0, os.path.abspath(os.path.join(os.pardir, 'scripts')))
+sys.path.insert(0, os.path.abspath(os.path.join(os.getcwd(), 'scripts')))
```

Comment:
If we are using the current directory, no need to use os.path.join
`sys.path.insert(0, os.path.abspath('scripts'))` should be fine